cmd/daemon/command/start.go

```diff
@@ -62,6 +81,7 @@ func StartDaemon() *cobra.Command {
 	command.Flags().StringVar(&authToken, "auth-token", os.Getenv("DAEMON_AUTH_TOKEN"), "token to be used to communicate with the release-manager")
 	command.Flags().StringVar(&environment, "environment", "", "environment where release-daemon is running")
 	command.Flags().StringVar(&kubeConfigPath, "kubeconfig", "", "path to kubeconfig file. If not specified, then daemon is expected to run inside kubernetes")
+	command.Flags().StringVar(&apiBinding, "api-binding", ":8080", "binding of the daemon api server")
```

Comment:
This flag name `api-binding` is not very informative into what it provides? It is for connecting `flux` with it, so maybe `flux-binding` or something like that?

Reply:
yeah, maybe "flux-api-binding"?

Reply:
Would be more descriptive yes 👍

Reply:
👍